src/test/features/directions-questionnaire/routes/support-required.ts

```diff
@@ -0,0 +1,185 @@
+import * as config from 'config'
+
+import { attachDefaultHooks } from 'test/routes/hooks'
+import 'test/routes/expectations'
+
+import { Paths } from 'directions-questionnaire/paths'
+import { Paths as ResponsePaths } from 'response/paths'
+import { Paths as ClaimantResponsePaths } from 'claimant-response/paths'
+import { Paths as DashboardPaths } from 'dashboard/paths'
+
+import { app } from 'main/app'
+
+import * as idamServiceMock from 'test/http-mocks/idam'
+import * as claimStoreServiceMock from 'test/http-mocks/claim-store'
+import * as draftStoreServiceMock from 'test/http-mocks/draft-store'
+import { checkAuthorizationGuards } from 'test/features/ccj/routes/checks/authorization-check'
+
+const claimWithDQ = {
+  ...claimStoreServiceMock.sampleClaimObj,
+  ...{ features: ['directionsQuestionnaire'] }
+}
+
+const externalId = claimStoreServiceMock.sampleClaimObj.externalId
+
+const cookieName: string = config.get<string>('session.cookieName')
+const defendantTaskListPage = ResponsePaths.taskListPage.evaluateUri({ externalId: externalId })
+const claimantTaskListPage = ClaimantResponsePaths.taskListPage.evaluateUri({ externalId: externalId })
+const pagePath = Paths.supportPage.evaluateUri({ externalId: externalId })
+
+function checkAccessGuard (app: any, method: string) {
+
+  it(`should redirect to dashboard page when DQ is not enabled for claim`, async () => {
+    idamServiceMock.resolveRetrieveUserFor('1', 'citizen')
+    claimStoreServiceMock.resolveRetrieveClaimByExternalId()
+    await request(app)[method](pagePath)
+      .set('Cookie', `${cookieName}=ABC`)
+      .expect(res => expect(res).to.be.redirect.toLocation(DashboardPaths.dashboardPage.uri))
+  })
+}
+
+describe('Directions Questionnaire - self witness page', () => {
+  attachDefaultHooks(app)
+
+  describe('on GET', () => {
+    const method = 'get'
+    checkAuthorizationGuards(app, method, pagePath)
+    checkAccessGuard(app, method)
+
+    context('when user authorised', () => {
+      beforeEach(() => {
+        idamServiceMock.resolveRetrieveUserFor('1', 'citizen')
+
+      })
+
+      it('should return 500 and render error page when cannot retrieve claims', async () => {
+        claimStoreServiceMock.rejectRetrieveClaimByExternalId('HTTP error')
+
+        await request(app)
+          .get(pagePath)
+          .set('Cookie', `${cookieName}=ABC`)
+          .expect(res => expect(res).to.be.serverError.withText('Error'))
+      })
+
+      it('should return 500 and render error page when cannot retrieve directions questionnaire draft', async () => {
+        claimStoreServiceMock.resolveRetrieveClaimByExternalId(claimWithDQ)
+        draftStoreServiceMock.rejectFind('Error')
+
+        await request(app)
+          .get(pagePath)
+          .set('Cookie', `${cookieName}=ABC`)
+          .expect(res => expect(res).to.be.serverError.withText('Error'))
+      })
+
+      it('should render page when everything is fine', async () => {
```

Comment:
Is it possible to add some more test for all the field available with label and checkbox under GET, please